code/api/src/modules/user/query.js

```diff
@@ -47,3 +47,9 @@ export const userGenders = {
   type: new GraphQLList(UserGenderType),
   resolve: getGenders
 }
+
+// user style
+export const userStyle = {
+  type: new GraphQLList(UserStyleType),
+  resolve: getStyle
+}
```

Comment:
Since style is an attribute of user, couldn't the FE just query user and request only the style? Since that function is already built we could just add the attribute.